gprimitives/src/lib.rs

```diff
@@ -28,7 +28,11 @@ pub use gear_ss58::Ss58Address;
 pub use primitive_types::{H160, H256, U256};
 
 mod macros;
+#[cfg(feature = "codec")]
+mod nonzero_u256;
 mod utils;
+#[cfg(feature = "codec")]
+pub use nonzero_u256::NonZeroU256;
```

Comment:
@breathx , @StackOverflowExcept1on can't we have types like this and U256, H256 w/o placing them under feature?

Reply:
we can do it in this PR